util/storage/mongo/conf.go

```diff
@@ -2,21 +2,22 @@ package mongo
 
 import (
 	"encoding/json"
+
 	"github.com/TykTechnologies/mserv/util/conf"
 )
 
-type MgoStoreConf struct {
-	ConnStr string
-	UseTLS  bool
-}
```

Comment:
Update https://github.com/TykTechnologies/mserv/blob/70ed7e215fb1215c8464532a78cd537838a737cb/mserv_example.conf#L13-L18 and https://github.com/TykTechnologies/mserv/blob/70ed7e215fb1215c8464532a78cd537838a737cb/mserv_slave.json#L15C1-L20 to remove the unused options like `UseTLS` and `Crypto`